Wikipedia Citation Templates.js

```diff
@@ -62,10 +65,14 @@ var typeMap = {
 	dictionaryEntry:"Cite encyclopedia"
 };
 
+// Most Wikipedias use the "First Last" name format in titles and citations.
+// This isn't universal, see e.g. Russian and Chinese.
 function formatAuthors(authors, useTypes) {
 	var text = "";
 	for each(var author in authors) {
-		text += ", "+author.firstName;
+		// The full "continuous" name uses no separators, which need be removed
+		// cf. "Luc, Jean André : de (1727-1817)"
```

Comment:
The date range should already be handled in the <a href="https://github.com/zotero/translators/blob/master/Primo.js#L277">Primo translator</a>. I tried this also in your primo instance and it worked there too. We could delete the remaining parenthesis on import. However, the ":" as seperation for the "de" looks for me quite unusual, cf. the <a href="http://viaf.org/viaf/103105362/">names in viaf</a>